cpp/src/parquet/column_reader.h

```diff
@@ -326,10 +333,13 @@ class RecordReader {
   int64_t levels_position() const { return levels_position_; }
 
   /// \brief Number of definition / repetition levels that have been written
-  /// internally in the reader
+  /// internally in the reader. This may be larger than values_written() because
+  // for repeated fields, we need to look at the levels in advance to figure out
```

Comment:
nit: comment style, continue 3-level comments (I believe these are special for doxygen).

Reply:
Done. I changed the comments a bit.